packages/components/src/components/DataAnalytics.tsx

```diff
@@ -32,30 +31,23 @@ const ButtonWrapper = styled.div`
     align-self: center;
 `;
 
-const StyledButton = styled(Button)`
-    min-width: 180px;
-`;
-
 const Label = styled.span`
     margin-left: ${spacingsPx.lg};
-    font-size: ${variables.FONT_SIZE.SMALL};
-    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
-    color: ${({ theme }) => theme.legacy.TYPE_DARK_GREY};
+    ${typography.hint};
+    color: ${({ theme }) => theme.textDefault};
```

Comment:
These are refactorings connected to updating color test (it had legacy colors, that we don't want to use especially in the test)